var/spack/repos/builtin/packages/lammps/package.py

```diff
@@ -0,0 +1,197 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+#
+# Please also see the LICENSE file for our notice and the LGPL.
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+import os
+import string
+
+
+class Lammps(Package):
+    """LAMMPS stands for Large-scale Atomic/Molecular Massively
+    Parallel Simulator."""
+    homepage = "http://lammps.sandia.gov/"
+    url      = "https://github.com/lammps/lammps/archive/stable_17Nov2016.tar.gz"
+
+    version('17Nov2016', '8aecc58a39f9775203517c62a592d13b',
```

Comment:
I would keep it as is. In general, we should probably name versions the same way the developer does. I'm trying to make all of the versions the same as is parsed in the URL so that `url_for_version` is unnecessary.